src/main/java/uk/gov/hmcts/ccd/domain/model/definition/AccessControlList.java

```diff
@@ -67,7 +76,7 @@ public AccessControlList duplicate() {
     @Override
     public String toString() {
         return "ACL{"
-            + "role='" + role + '\''
```

Comment:
Intentionally deleted?

Reply:
Yes

Reply:
this will be removed once all the test cases are migrated